include/RAJA/index/IndexSet.hpp

```diff
@@ -60,7 +60,7 @@ namespace indexset
 /// over segments.  The second describes the policy for executing
 /// each segment.
 ///
-template <typename SEG_ITER_POLICY_T, typename SEG_EXEC_POLICY_T>
+template <typename SEG_ITER_POLICY_T, typename SEG_EXEC_POLICY_T = void>
```

Comment:
I don't understand this. What is this for? Does it introduce potential issues for index set users?

Reply:
This solved a resolution issue in the CHAI execution space thingy.

It shouldn't be a problem for users of IndexSets because you wont be able to compile a forall<> if the segexec is void